infrastructure/modules/products/app-registrations.tf

```diff
@@ -0,0 +1,16 @@
+resource "azuread_application" "pars-upload" {
+  name                    = "pars-upload-${var.namespace}"
+  reply_urls              = ["http://localhost:3000", "https://${var.namespace}"] #this url needs to be correct for the environment
```

Comment:
@rjkerrison @anubha22  we should be able to know this url after we'd done the terraform deploy part for the website

Reply:
will we want different reply_urls for different environments?

Reply:
yes, we could put them all in one array but we might as well have that array come from an environmental variable instead so that dev can have localhost + the deployed dev url and non prod and prod can just have their own web url

Reply:
in fact I'm gonna do that now in this PR and we can replace the values when we know them